src/server/api/routers/map_mod_publisher/mod.ts

```diff
@@ -477,6 +507,52 @@ const getGamebananaModInfo = async (gamebananaModID: number): Promise<Gamebanana
 
 
 
+const getModSearchDatabaseInfoForMod = async (gamebananaModID: number): Promise<ModSearchDatabase_ModInfo_NoId_Partial> => {
+    const modSearchDatabase = await getCurrentModSearchDatabase();
+
+
+    let modInfo: ModSearchDatabase_ModInfo_NoId_Partial | undefined = undefined;
+
+    for (const mod of modSearchDatabase) {
+        if (mod.GameBananaId === gamebananaModID) {
+            modInfo = {
```

Comment:
Maybe return the object here, return an empty object at the end of the function and remove the modInfo variable?

Reply:
Oh yeah, that makes more sense